bentoml/__init__.py

```diff
@@ -34,6 +37,10 @@
     save,
 )
 
+from bentoml.cli import create_bentoml_cli
+
+commandline_interface = create_bentoml_cli()
```

Comment:
What's the reason for moving it up here?

Reply:
This is because we need `inject_dependencies` to happen before the creation of CLI.